x-pack/plugins/infra/public/pages/metrics/hosts/constants.ts

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { TabId } from './types';
+
+export const tabIds: Record<string, TabId> = {
```

Comment:
It seems that this is only used in `tabs.tsx`, I'm curious if it could be moved there, instead of this in this file.

Side note: 
You'll probably get advantages from autocomplete in `tabs.tsx`, if you define it like this, letting TS infer type :

```ts
const tabIds = {
  ALERTS: 'alerts',
  METRICS: 'metrics',
};
```

or as an enum (enums could probably be in the new `types.ts` file)

```ts
enum TabIds {
  ALERTS: 'alerts',
  METRICS: 'metrics',
};
```

Reply:
Agree, this has been something I moved a couple of times due to refactors and code organization, but your suggestion is the right choice, thanks! :)